### PR TITLE
chore(ctl)!: update ctl to 0.31.0

### PR DESCRIPTION
**Pull request by brooksmtownsend**
Signed-off-by: Brooks Townsend <brooksmtownsend@gmail.com>

## Feature or Problem
This PR updates the control interface to 0.31.0.

## Related Issues
N/A

## Release Information
v0.80.0

## Testing

Built on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [x] aarch64-darwin
- [ ] x86_64-windows

Tested on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [x] aarch64-darwin
- [ ] x86_64-windows

### Manual Verification
Tested with a few wadm deploys + queries with wash to ensure direct claims and link queries worked well.

### CODE REVIEW

**Review by rvolosatovs — COMMENTED** [review]
Can we please get #807 merged first? Otherwise merge of this PR will require another merge from `main` into the PR

**Review by rvolosatovs — APPROVED** [review]
As a follow-up, can we move the crate itself into the workspace?

**Comment by rvolosatovs** [comment]
@brooksmtownsend this will now require updates in wash as well, could you take care of that?

**Review by connorsmith256 — APPROVED** [review]

File: tests/wasmbus.rs [hunk 1]
```diff
@@ -573,7 +571,6 @@ async fn wasmbus() -> anyhow::Result<()> {
                 js_domain,
                 labels,
                 lattice_prefix,
-                prov_rpc_host,
```
[the remark this answers is not in the document]
Comment (brooksmtownsend) [reply tests/wasmbus.rs:576]: Hm good point. We probably should remove that field entirely, if `None` is an acceptable outcome this won't break things 🤔 I'm not aware of any consumers of the lattice controller that depend on that field and I think we generally unwrap_or with the rpc host

**Comment by brooksmtownsend** [comment]
@rvolosatovs brought over a few commits from wash to add the work in both places. Monorepo changes in one PR FTW!

**Review by rvolosatovs — APPROVED** [review]

**Review by connorsmith256 — APPROVED** [review]

**Pull request merged**